openmdao/drivers/test/test_pyoptsparse.py

```diff
@@ -131,8 +130,8 @@ def test_simple_paraboloid_equality(self):
         prob.run()
 
         # Minimum should be at (7.166667, -7.833334)
-        assert_rel_error(self, prob['x'], 7.16667, 1e-6)
-        assert_rel_error(self, prob['y'], -7.833334, 1e-6)
+        assert_rel_error(self, prob['x'], 7.16667, 1e-3)
```

Comment:
Hm, the question I would have is why didn't it get anywhere close to that. (I assume you changed the test tolerance to 1e-3 because 1e-4 wasn't even good enough.)  If you set it to 1e-9, does the test match to 1e-6?